src/routes/user/[user]/+page.svelte

```diff
@@ -27,14 +27,17 @@
 	let completions: MissionCompletion[] = data.completions;
 	let tpMissions: Mission[] = data.tpMissions;
 
+	const dateOptions: Intl.DateTimeFormatOptions = { year: 'numeric', month: 'short', day: 'numeric' };
+	// let seeUnverif = $page.data.user !== null && $page.data.user.id == shownUser?.id || hasPermission($page.data.user, Permission.RenameUser);
+	let seeUnverif = $page.data.user !== null && $page.data.user.id == shownUser?.id;
```

Comment:
These don't seem to be used.

Reply:
These will be used soon, removed for now, thanks.